Mathlib/RingTheory/LocalRing/Quotient.lean

```diff
@@ -67,7 +67,7 @@ theorem finrank_quotient_map :
   · let b := Module.Free.chooseBasis (R ⧸ p) (S ⧸ pS)
     choose b' hb' using fun i ↦ Ideal.Quotient.mk_surjective (b i)
     conv_rhs => rw [finrank_eq_card_chooseBasisIndex]
-    apply finrank_le_of_span_eq_top
+    refine finrank_le_of_span_eq_top (v := b') ?_
```

Comment:
In case anyone wonders, `apply ... (v := b')` would have been enough, but I thought that using `refine` was "better form".  The reason for passing the explicit argument is that it created a side goal that was automatically closed once the "main" branch got closed.  The assumption `b'` is a function, so maybe that explains why the goal was delayed.

Reply:
I wondered, thanks for explaining.